test/sharness/t0260-sharding.sh

```diff
@@ -129,12 +147,12 @@ test_list_incomplete_dir() {
 
   test_expect_success "can list part of the directory" '
     ipfs ls "$largeSHA3dir" 2> ls_err_out
-    echo "Error: merkledag: not found" > exp_err_out &&
+    echo "Error: failed to fetch all nodes" > exp_err_out &&
```

Comment:
@aschmahmann Please verify if this change is correct.

Reply:
This seems fine. AFAICT it was triggered by switching from Get to GetMany in https://github.com/ipfs/go-unixfs/pull/111 since Get uses https://github.com/ipfs/go-merkledag/blob/bfaa7663d177de206edb8a513b09ab4f94b95e57/merkledag.go#L83 while GetMany uses https://github.com/ipfs/go-merkledag/blob/bfaa7663d177de206edb8a513b09ab4f94b95e57/merkledag.go#L258

Reply:
Makes sense.